app/src/main/java/org/simple/clinic/appconfig/CountryModule.kt

```diff
@@ -40,4 +42,26 @@ object CountryModule {
       appConfigRepository.saveDeployment(deployment)
     }
   }
+
+  private fun createV2CountryIfOldCountryIsPresent(appConfigRepository: AppConfigRepository) {
+    val selectedOldCountry = appConfigRepository.currentCountry().toNullable()
+    val selectedNewCountry = appConfigRepository.currentCountryV2()
+
+    if (selectedOldCountry != null && selectedNewCountry == null) {
+      // Since V1 country doesn't have deployment names, we are going with country name
+      val deployment = Deployment(
+          displayName = selectedOldCountry.displayName,
+          endPoint = selectedOldCountry.endpoint
+      )
+      val country = CountryV2(
+          isoCountryCode = selectedOldCountry.isoCountryCode,
+          displayName = selectedOldCountry.displayName,
+          isdCode = selectedOldCountry.isdCode,
+          deployments = listOf(deployment)
```

Comment:
Hmm, for a given installation of the app, we will never need to store all the deployments, just the one the app needs to communicate to the server.

I think we can simplify this by having the `CountryPayload` store a list of deployments and only have a single `Deployment` for each `Country` saved in the local repository when we complete the state and deployment selection. This would mean that we don't need to inject two different things at usage sites and can continue just injecting a single `Country` instance that has a reference to the `Deployment`.

What do you think?

Reply:
Yeah, I am thinking about it. In my mind, I think I liked having a different `Deployment` being injected in only sync APIs, and use `Country` for the rest of the areas. But obviously like you suggested we can just place one deployment after state selection. We can revisit this once state selection is done. I think this should be a straightforward change.

Reply:
Yeah, I think it will also simplify the migration of the `Country` and enforce safety since we won't be able to save a `Country` separately from a deployment.

Reply:
Cool, will make this change after state selection is done. For now creating a story for it.